Dockerfile-default-rocm

```diff
@@ -1,40 +1,124 @@
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE}
 
+RUN apt remove -y openmpi ucx
+#Let's move any existing /opt/ompi out of the way
+RUN mv /opt/ompi /opt/ompi.0 
```

Comment:
why not just remove it? it is possible to still make the existing `/opt/ompi` work somehow?